### PR TITLE
Added ability to suppress cron output

### PR DESCRIPTION
**Pull request by grega**
Due to `--quiet` no longer being supported (see: https://github.com/voxpupuli/puppet-letsencrypt/pull/74) this provides a way to silence cron-generated email (the client still logs to `/var/log/letsencrypt/letsencrypt.log`).

This seems to be the cleanest way to do so (other methods of disabling cron email has a tendency to affect other commands as well).

### CODE REVIEW

**Comment by grega** [comment]
Recent commit tests failed due to Rubocop upsets - have fixed.

File: README.md [hunk 1]
```diff
@@ -117,13 +117,15 @@ letsencrypt::certonly { 'foo':
 To automatically renew a certificate, you can pass the `manage_cron` parameter.
 You can optionally add a shell command to be run on success using the `cron_success_command` parameter.
 You can optionally add a shell command to be run on before using the `cron_before_command` parameter.
+You can disable emails generated by the cron command using the `disable_cron_emails` parameter.
```
Comment (bastelfreak) [inline README.md]: should we rephrase this? We don't disable cron emails, we suppress the daemon output.

Comment (grega) [inline README.md]: @bastelfreak yes that's better phrasing.

**Comment by grega** [comment]
> should we rephrase this? We don't disable cron emails, we suppress the daemon output.

The latest commit rephrases to use the terminology "Suppress cron output"

(seems to make sense to keep `cron` in since we're dealing with the `manage_cron` functionality of the module)

File: .rubocop.yml [hunk 2]
```diff
@@ -302,7 +302,7 @@ Style/EmptyLiteral:
 Metrics/LineLength:
   Enabled: False
 
-Style/MethodCallParentheses:
+Style/MethodCallWithoutArgsParentheses:
```
Comment (bastelfreak) [inline .rubocop.yml]: please rebase against master and drop changes to this file.

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -24,8 +24,8 @@
     puppetversion: Puppet.version,
     facterversion: Facter.version
   }
-  default_facts.merge!(YAML.load(File.read(File.expand_path('../default_facts.yml', __FILE__)))) if File.exist?(File.expand_path('../default_facts.yml', __FILE__))
-  default_facts.merge!(YAML.load(File.read(File.expand_path('../default_module_facts.yml', __FILE__)))) if File.exist?(File.expand_path('../default_module_facts.yml', __FILE__))
+  default_facts.merge!(YAML.safe_load(File.read(File.expand_path('../default_facts.yml', __FILE__)))) if File.exist?(File.expand_path('../default_facts.yml', __FILE__))
```
Comment (bastelfreak) [inline spec/spec_helper.rb]: same here. please rebase against master and drop changes to this file. We fixed the rubocop issues in the master branch so no changes should be needed here.

**Comment by grega** [comment]
@bastelfreak have rebased & performed a general tidy/squash of the commits 👍

**Comment by bastelfreak** [comment]
thanks!

**Pull request merged**